tests/integration/test_non_default_compression/test.py

```diff
@@ -244,3 +251,59 @@ def test_uncompressed_cache_plus_zstd_codec(start_cluster):
         )
         == "10000\n"
     )
+
+def test_preconfigured_deflateqpl_codec(start_cluster):
```

Comment:
For weird reason, no code in this test interacts with "node6" . Could we make QPL node6 and rename existing node6 to node7? Just want to make sure that the weird, orphaned node6 is listed last.

Reply:
done